projects/core/constants/notification-status-icons.ts

```diff
@@ -0,0 +1,7 @@
+/** Icon names by status **/
+export const STATUS_ICONS = {
```

Comment:
Do we need it anywhere other than options token? If so, we can just put it there and not export.

Reply:
I wanted to use this in tests, but I didn't figure out how to properly test a new api.